README.md

```diff
@@ -1 +1,111 @@
-# CMS_Public_Use_File
+# CMS Public Use File:<br/>A Machine Learning Case Study
+
+## Table of Contents
+- [Introduction](#introduction)
```

Comment:
Table of content can be created via these "links". The text inside the brackets `[]` is the text that will be displayed. The text inside the parenthesis `()` is the title of where the link will go. *Notice the link is in lower case and spaces are represented as hyphens `-`.